wp_api_integration_tests/tests/test_cookie_authentication_immut.rs

```diff
@@ -0,0 +1,70 @@
+use wp_api::{WpApiClient, WpAuthentication, WpLoginCredentials};
+
+use wp_api_integration_tests::*;
+
+use serial_test::serial;
+
+#[tokio::test]
+#[serial]
```

Comment:
```suggestion
#[parallel]
```

The reason we use `#[serial]` in mutable tests is because the execution of a mutable test can interfere with other tests - both mutable and immutable ones. However, the same doesn't apply for immutable tests, and it's safe to run them in parallel.